python/sub_data.py

```diff
@@ -162,47 +177,44 @@ def on_new_pow_data(self, *args, **kwargs):
         data = kwargs.get('data')
         print('pow data: {}'.format(data))
 
+    # callbacks functions
+    def on_create_session_done(self, *args, **kwargs):
+        print('on_create_session_done')
+
+        # subribe data 
+        self.sub(self.streams)
 
 # -----------------------------------------------------------
 # 
 # SETTING
-#   - replace your license, client_id, client_secret to user dic
-#   - specify infor for record and export
-#   - connect your headset with dongle or bluetooth, you should saw headset on EmotivApp
-# SUBSCRIBE
-#     you need to folow steps:
-#         1) do_prepare_steps: for authorization, connect headset and create working session.
-#         2) sub(): to subscribe data, you can subscribe one stream or multiple streams
+#   - replace your  client_id, client_secret to user dic
+#   - connect your headset with dongle or bluetooth, you should saw headset on Emotiv Launcher
 # RESULT
 #   - the data labels will be retrieved at on_new_data_labels
 #   - the data will be retreived at on_new_[dataStream]_data
 # 
 # -----------------------------------------------------------
 
 """
-    client_id, client_secret:
-    To get a client id and a client secret, you must connect to your Emotiv account on emotiv.com and create a Cortex app
-    To subscribe eeg you need to put a valid licese (PRO license)
+    client_id, client_secret: required params
```

Comment:
Instead of describing everything in the example, it's better to have gitbook link here

Reply:
yes, i will update